### PR TITLE
Fix logic operands + Twitter error response message

### PR DESCRIPTION
**Pull request by adam-riha**
Hi Felix,

this PR fixes logical operators error and TwitterException.php

- convert logical operators to uppercase
  ```
  Twitter error response: 
  Ambiguous use of or as a keyword. Use OR to logically join two clauses, or "or" to find occurrences of or in text
  ```
  
- Twitter is probably no longer attaching an error ID, but instead a `type` to the response. 
I also kept the ID if it was part of the response. 

  ```php
  // TwitterException::fromPayload
  
  $payload = [
    "value" => "__RULE_HERE__"
    "details" => array:1 [
      0 => "Ambiguous use of or as a keyword. Use OR to logically join two clauses, or "or" to find occurrences of or in text (at position 25)"
    ]
    "title" => "UnprocessableEntity"
    "type" => "https://api.twitter.com/2/problems/invalid-rules"
  ]
  ```

### CODE REVIEW

**Comment by felixdorn** [comment]
Thanks for contributing ! Can you please update the tests accordingly?

There's no contributor guide but you'll also need to regenerate the tests by running `composer gen:tests`.

File: src/Exceptions/TwitterException.php [hunk 1]
```diff
@@ -34,7 +34,9 @@ public static function fromPayload(array $payload): TwitterException
             // For most errors related to rule creation, details are provided.
             $hasDetails = array_key_exists('details', $error);
 
-            $buffer .= sprintf("[%s] %s %s [%s]\n", $error['title'], $error['value'], $hasDetails ? ': ' . implode('; ', $error['details']) : '', $error['id']);
+            $errorIdOrType = $error['id'] ?? ($error['type'] ?? '');
```
Comment (felixdorn) [inline src/Exceptions/TwitterException.php:37]: If there's no ID or type, that would leave empty brackets on the error message. To solve that, perhaps change the default value to "unknow error" or something.

It shouldn't happen but, Twitter also returns 2xx on errors so what do we know...

**Comment by adam-riha** [comment]
Sure thing.

----
By the way I have a problem with the stream, I can't receive any data from Twitter, even with a very general rule.
I don't know the reason yet I'm trying to debug it.

**Comment by felixdorn** [comment]
Sometimes it takes like 30 seconds before you receive something.

**Comment by felixdorn** [comment]
I've released 1.0.2, lmk if you're having any more issues!

**Pull request merged**